read_test.go

```diff
@@ -595,6 +595,78 @@ func TestReadPixelData_SkipPixelData(t *testing.T) {
 	}
 }
 
+// Returns a fake DICOM group 2 header with the FileMetaInformationGroupLength tag missing (0x0002,0x0000)
+func fakeHeader_NoFileMetaInformationGroupLength() *bytes.Buffer {
```

Comment:
nit: no need for the underscore for this helper function, can do something like (and do something similar to function below). 

More on this: https://google.github.io/styleguide/go/decisions#underscores

```suggestion
func headerWithNoFileMetaInformationGroupLength() *bytes.Buffer {
```

Reply:
Done.  Used suggested names.